app/src/features/settings/components/BillingTeam/components/BillingDetails/modals/ReviewJoinRequestModal/ReviewJoinRequestModal.tsx

```diff
@@ -19,6 +17,18 @@ interface ReviewJoinRequestModalProps {
   requestAction: BillingTeamJoinRequestAction;
 }
 
+enum ReviewResultCodes {
```

Comment:
We should rename this to `ReviewResultMessage`, also enum should always be singular.